src/Illuminate/Queue/Worker.php

```diff
@@ -151,10 +151,13 @@ protected function registerTimeoutHandler($job, WorkerOptions $options)
         // process if it is running too long because it has frozen. This uses the async
         // signals supported in recent versions of PHP to accomplish it conveniently.
         pcntl_signal(SIGALRM, function () use ($job, $options) {
-            $this->markJobAsFailedIfWillExceedMaxAttempts(
-                $job->getConnectionName(), $job, (int) $options->maxTries, $this->maxAttemptsExceededException($job)
-            );
-
+            // Check if the job exists in queue.
```

Comment:
We can use first return instead of one more indention level.
```php
if (! $job) {
    return;
}
```

Reply:
we have to call $this->kill(1) next to the if statement and we will have to mark job as failed before killing worker so i need to use one more indentation level.